pages/unix-epoch-utc-conversion.test.tsx

```diff
@@ -61,15 +59,6 @@ async function expectUtcError(
 }
 
 describe(UnixEpochUtcConversion.name, () => {
-  it('displays correct unix epoch time', async () => {
```

Comment:
why delete test case?

Reply:
I decided that this test was not useful at all. To make it work we need: either mock the fn. returns (so it loses sense here), or match it against a range, I find it rather obscure.